src/widgets/FridgeDisplay.js

```diff
@@ -57,6 +57,25 @@ export const FridgeDisplayComponent = ({
     []
   );
 
+  const ChartArea = () =>
+    (minLine.length && (
+      <VaccineChart
+        minLine={minLine}
+        maxDomain={maxDomain}
+        minDomain={minDomain}
+        maxLine={maxLine}
+        breaches={breaches}
+        onPressBreach={onOpenBreachModal}
+        xTickFormat={temperatureFormatter}
+      />
+    )) || <NoLogsComponent />;
+
+  const Spinner = () => (
+    <FlexView justifyContent="center" flex={1} alignItems="center">
+      <ActivityIndicator size="small" color={SUSSOL_ORANGE} />
+    </FlexView>
```

Comment:
just tidying the render method a bit - it's slightly different and actually shows the no logs component now..! Not memoizing as it isn't really needed